executables/collector/src/main/java/de/cyface/collector/verticle/CollectorApiVerticle.java

```diff
@@ -115,8 +116,14 @@ public void start(final Promise<Void> startFuture) throws Exception {
         final var configV3 = new Config(vertx);
 
         // Create indices
+        final var unique = new IndexOptions().unique(true);
         final var measurementIndex = new JsonObject().put("metadata.deviceId", 1).put("metadata.measurementId", 1);
-        final var indexCreation = configV3.getDataDatabase().createIndex("fs.files", measurementIndex);
+        // While supporting `v2.MeasurementHandler` we must support multiple entries per did/mid (because of `fileType`)
```

Comment:
I guess we should remove that as soon as iOS is updated in the store.

Reply:
Had the same thought, but than I remembered, that it's not decided yet that SR ports to the new API, so we still need the V2 Collector maintained for debugging.